rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -684,6 +686,14 @@ public String corsDomains() {
         return configFromFiles.getString(PROPERTY_RPC_CORS);
     }
 
+    public int getMaxPayloadSize() {
```

Comment:
```suggestion
    public int rpcHttpMaxAggregatedFrameSize() {
```
similar to `rpcWebSocketMaxAggregatedFrameSize()` ?